packages/core/src/components/ModalNew/Modal/Modal.types.tsx

```diff
@@ -5,6 +5,7 @@ import { ModalTopActionsProps } from "../ModalTopActions/ModalTopActions.types";
 export type ModalSize = "small" | "medium" | "large";
 
 export interface ModalProps extends VibeComponentProps {
+  id: string;
```

Comment:
Why is it now mandatory? Also, it won't guarantee uniqueness as users can pass same id to multiple modals

Reply:
because I don't have `useId` in React 16 😅
I wanted to generate an id for title and subtitle which are:
```
[my-modal-id]_label
[my-modal-id]_desc
```
I can just implement some generate id logic in ModalHeader as a substitute. wdyt?

Reply:
Can it have a default without an id and if id is provided - use it for these ids?